hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -5943,6 +5943,9 @@ LocatedBlock bumpBlockGenerationStamp(ExtendedBlock block,
     }
     // Ensure we record the new generation stamp
     getEditLog().logSync();
+    LOG.info("bumpBlockGenerationStamp(" + locatedBlock.getBlock()
+        + ", client=" + clientName
```

Comment:
Thanks for your contribution, but we need to use {} instead of +.

like this
```
 LOG.info("bumpBlockGenerationStamp({}), client={}", locatedBlock.getBlock(), clientName)
```

Reply:
@slfan1989 , thanks for your review. I will fix it.